pkg/virt-launcher/virtwrap/api/testdata/domain_x86_64.xml.tmpl

```diff
@@ -66,6 +66,7 @@
     </kvm>
     <pvspinlock state="off"></pvspinlock>
     <pmu state="off"></pmu>
+    <vmport state="off"></vmport>
```

Comment:
you should also modify the pkg/virt-launcher/virtwrap/converter/testdata/domain_x86_64_root.xml.tmpl template.